include/drivers/memc/memc_mcux_flexspi.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <sys/types.h>
+#include <fsl_flexspi.h>
+
+int memc_flexspi_update_lut(const struct device *dev, uint32_t index,
```

Comment:
Are these APIs meant for application level use?

Reply:
No

Reply:
Wouldn't the following make more sense in `drivers/flash/CMakeLists.txt`?
```
zephyr_include_directories_ifdef(
  CONFIG_FLASH_MCUX_FLEXSPI_NOR
  ${ZEPHYR_BASE}/drivers/memc
)
```